OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs

```diff
@@ -468,6 +468,8 @@ public MappingProfile()
                    .ForMember(dest => dest.Email, opt => opt.MapFrom(src => src.User.Email))
                    .ForMember(dest => dest.CATOTTGCategory, opt => opt.MapFrom(src => src.CATOTTG.Category))
                    .ForMember(dest => dest.CATOTTGName, opt => opt.MapFrom(src => src.CATOTTG.Name))
+                   .ForMember(dest => dest.RegionId, opt => opt.MapFrom(src => src.CATOTTG != null ? src.CATOTTG.Parent != null ? src.CATOTTG.Parent.Parent != null ? src.CATOTTG.Parent.Parent.Id :0 :0 :0))
```

Comment:
(Code style) I might be wrong but could you try instead of lot of ternary operators use elvis and null-coalescing operators, like
```suggestion
                   .ForMember(dest => dest.RegionId, opt => opt.MapFrom(src => src.CATOTTG?.Parent?.Parent ?? 0))
```

Reply:
Using such case I`ve caught the following error:
"An expression tree lambda cannot contain conditional access expressions".
Developers on StackOverflow suggest using something similar to what I tried to realize

Reply:
Thanks for sharing! (Was kinda expecting this but wanted to clarify)